acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java

```diff
@@ -613,10 +615,11 @@ public void validate(PlainAccessResource plainAccessResource) {
 
         // Check the white addr for accessKey
         String aclFileName = accessKeyTable.get(plainAccessResource.getAccessKey());
-        PlainAccessResource ownedAccess = aclPlainAccessResourceMap.get(aclFileName).get(plainAccessResource.getAccessKey());
-        if (null == ownedAccess) {
+        if (aclPlainAccessResourceMap.get(aclFileName) == null ||
+                aclPlainAccessResourceMap.get(aclFileName).get(plainAccessResource.getAccessKey()) == null) {
```

Comment:
You'd better extract `aclPlainAccessResourceMap.get(aclFileName)` as a variable so that the code would be more clear.

Reply:
thanks for your suggestion, i will adjust it